docker/etna-base-dev/entrypoints/puma.rb

```diff
@@ -0,0 +1,7 @@
+
+activate_control_app
+plugin :yabeda
+plugin :yabeda_prometheus
```

Comment:
https://github.com/yabeda-rb/yabeda-prometheus and https://github.com/yabeda-rb/yabeda-puma-plugin work together to support automatic generation of puma webserver metrics that we can pull!